pkg/lib/aws/cloudwatch.go

```diff
@@ -266,6 +315,13 @@ func (grid *CloudWatchWidgetGrid) AddWidget(
 			},
 		},
 	)
+
+	return nil
+}
+
+// GetWidgets gets the list of CloudWatchWidget widgets
+func (grid *CloudWatchWidgetGrid) GetWidgets() []CloudWatchWidget {
```

Comment:
What was your motivation for adding the `GetWidgets()` function?

Reply:
Just because it's using a getter as opposed to accessing the field directly. Do you think that's okay or should it be reverted?